update.sh

```diff
@@ -2,7 +2,7 @@
 
 # 2.3.0 2.4.0 2.5.0 のサポートは終了しました
 # 3.0.0〜5.2.0 はDebian 10で凍結し、update反映外とします
-versions=(5.3.0)
+versions=(5.5.0)
```

Comment:
```suggestion
versions=(5.3.0 5.4.0 5.5.0)
```
rootのDockerfileを反映させるバージョンは上記3つが対象が正しいですかね？(コメントより